ui/src/app/views/invitation-key/component.spec.ts

```diff
@@ -0,0 +1,102 @@
+import {DebugElement} from '@angular/core';
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {FormsModule} from '@angular/forms';
+import {UrlSegment} from '@angular/router';
+import {RouterTestingModule} from '@angular/router/testing';
+
+import {ClarityModule} from 'clarity-angular';
+import {ProfileService} from 'generated';
+import {ErrorHandlingServiceStub} from 'testing/stubs/error-handling-service-stub';
+import {ProfileServiceStub} from 'testing/stubs/profile-service-stub';
+
+import {
+    queryByCss, simulateClick,
+    updateAndTick
+} from '../../../testing/test-helpers';
+import {ErrorHandlingService} from '../../services/error-handling.service';
+import {SignInService} from '../../services/sign-in.service';
+import {AccountCreationComponent} from '../account-creation/component';
+import {InvitationKeyComponent} from '../invitation-key/component';
+
+
+
```

Comment:
I don't think this pattern (lots of space after imports) exists anywhere else in our code. Let's knock out a couple of these newlines.